andela/hc-shleemypants
# Ft add unresolved issues 155965538

**What does this PR do ?**
new dashboard tab for unresolved stuff

**Description of tasks to be completed**
the user should have a separate dashboard view of their failed jobs

How should this be manually tested?
* check out project code:

        $ git clone https://github.com/healthchecks/healthchecks.git

* install requirements (Django, ...) into virtualenv:

        $ pip install -r healthchecks/requirements.txt


What are the relevant pivotal tracker stories?

[155965538](https://www.pivotaltracker.com/story/show/155965538)
